tests/test_api.py

```diff
@@ -100,66 +100,21 @@ def test_query_handle(self, das: DistributedAtomSpace):
 
         ret = das.query(expression, {'return_type': QueryOutputFormat.HANDLE})
 
-        expected_values = [
```

Comment:
It seems you are removing old tests and not adding new ones. In particular I see no test for the wrapping itself.

Reply:
No. I removed this part because it was wrong. Changing the return, the approach was wrong. I'll add tests for wrapping yes